File: src/lolicon.pl
#!/usr/bin/env perl

use strict;
use warnings;

sub intice {
    #Nothing because weebs are already attracted to underage anime girls
}

sub call_the_fbi {
    if ($_[0] < 15) {
        my $output = qx/call_911/;
        print $output;
    } else {
        warn "Probably just something weird happened.";
    }
}

sub main {
    print "What's the girl's age? ";
    my $age = <STDIN> + 0;
    call_the_fbi($age);
}

main();

File: src/lolicon.scala
import scala.sys.process._

object lolicon {
  def intice(): Int = {
    // Nothing because weebs are already attracted to underage anime girls
    return 0
  }


  def main(args: Array[String]): Unit = {
    println("What\'s the girl\'s age?")
    var age = scala.io.StdIn.readInt()
    callthefbi(age)
  }

  def callthefbi(age:Int): Int = {
    if (age < 15){
      val cal911 = "call_911"
      val output = cal911.!!
      println(output)
    } else {
      println("Probably just something weird happened")
    }
    return 0
  }
}
